SECURITY.md

```diff
@@ -0,0 +1,67 @@
+# Security Policy
+
+Being a security tool itself, the security of Suricata itself is naturally of
+great importance. This document describes the policy around security issues.
+
```

Comment:
Some more welcoming word, like if you find a security vulnerability, please tell us ?

Reply:
sure, we can add something like that